event_routing_backends/tasks.py

```diff
@@ -134,5 +136,7 @@ def bulk_send_events(task, events, router_type, host_config):
             ),
             exc_info=True
         )
+        if not task:
```

Comment:
A comment about what's happening here would be good helpful, also marking `task` in the docstring as optional.